test/bad_dtls_test.c

```diff
@@ -548,8 +548,9 @@ static int test_bad_dtls(void)
 
         if (!TEST_true(send_record(rbio, SSL3_RT_APPLICATION_DATA, tests[i].seq,
                                    &tests[i].seq, sizeof(uint64_t)))) {
-            TEST_error("Failed to send data seq #0x%lx (%d)\n",
-                       tests[i].seq, i);
+            TEST_error("Failed to send data seq #0x%08x%08x (%d)\n",
+                       (uint32_t)(tests[i].seq >> 32),
+                       (uint32_t)(tests[i].seq & 0xFFFFFFFF), i);
```

Comment:
> Updating to %llx didn't work under 64-bit

Well, this does *work* [and even so to say universally] in sense that if you *execute* code passing %llx to printf, it will actually produce sensible result. It's the *warning* that is in your way, isn't it?